dask/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: dask
-version: 0.17.1
+version: 0.17.2
```

Comment:
`0.17.1` is the latest dask version so this probably should remain fixed. Do helm chart versions actually mean anything or does the latest always get pulled...?

Reply:
Apparently there is a separate appVersion that we'll eventually start using.  I plan to bump this up to 1.0.0 once we get the chart into stable.